File: csharp/high-scores/HighScores.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class HighScores
{
    private List<int> scores;

    public HighScores(List<int> list)
    {
        this.scores = list;
    }

    public List<int> Scores()
    {
        return this.scores;
    }

    public int Latest()
    {
        return this.scores.Last();
    }

    public int PersonalBest()
    {
        return this.scores.Max();
    }

    public List<int> PersonalTopThree()
    {
        return this.scores.OrderByDescending(x => x).Take(3).ToList();
    }
}

File: python/high-scores/high_scores.py

def latest(scores):
    return scores[len(scores) - 1]


def personal_best(scores):
    max = scores[0]
    for value in scores:
        if max < value:
            max = value
    return max


def personal_top_three(scores):
    scores.sort(reverse=True)
    del scores[3: len(scores)]
    return scores
